packages/rocketchat-ui-message/client/message.html

```diff
@@ -6,7 +6,7 @@
 			{{else}}
 				<button class="thumb user-card-message" data-username="{{u.username}}" tabindex="1">
 					<div class="avatar">
-						<div class="avatar-image" style="background-image:url({{avatar}});"></div>
```

Comment:
@engelgabriel I don't know now, cuz we save usernames there too, can be a problem